cellprofiler/modules/classifyobjects.py

```diff
@@ -247,110 +254,113 @@ def add_single_measurement(self, can_delete=True):
 
         def object_fn():
             return group.object_name.value
 
         group.append("measurement", cps.Measurement(
-                "Select the measurement to classify by", object_fn, doc="""
-            Select a measurement made by a previous module. The objects
-            will be classified according to their values for this
-            measurement."""))
+                "Select the measurement to classify by", object_fn, doc="""\
+Select a measurement made by a previous module. The objects will be
+classified according to their values for this measurement.
+"""))
 
         group.append("bin_choice", cps.Choice(
                 "Select bin spacing",
-                [BC_EVEN, BC_CUSTOM], doc="""
-            Select how you want to define the spacing of the bins.
-            You have the following options:
-            <ul>
-            <li><i>%(BC_EVEN)s:</i> Choose this if you want to specify
-            bins of equal size, bounded by upper and lower limits. If you
-            want two bins, choose this option and then provide a single
-            threshold when asked.</li>
-            <li><i>%(BC_CUSTOM)s:</i> Choose this option to create the
-            indicated number of bins  at evenly spaced intervals between the
-            low and high threshold.
-            You also have the option to create bins for objects that fall below
-            or above the low and high threshold.</li>
-            </ul>
-            """ % globals()))
+                [BC_EVEN, BC_CUSTOM], doc="""\
+Select how you want to define the spacing of the bins. You have the
+following options:
+
+-  *%(BC_EVEN)s:* Choose this if you want to specify bins of equal
+   size, bounded by upper and lower limits. If you want two bins, choose
+   this option and then provide a single threshold when asked.
+-  *%(BC_CUSTOM)s:* Choose this option to create the indicated number
+   of bins at evenly spaced intervals between the low and high
+   threshold. You also have the option to create bins for objects that
+   fall below or above the low and high threshold.
+""" % globals()))
 
         group.append("bin_count", cps.Integer(
-                "Number of bins", 3, minval=1, doc="""
-            This is the number of bins that will be created between
-            the low and high threshold"""
+                "Number of bins", 3, minval=1, doc="""\
+This is the number of bins that will be created between
+the low and high threshold"""
         ))
 
         group.append("low_threshold", cps.Float(
-                "Lower threshold", 0, doc="""
-            <i>(Used only if "%(BC_EVEN)s" selected)</i><br>
-            This is the threshold that separates the lowest bin from the
-            others. The lower threshold, upper threshold, and number of bins
-            define the thresholds of bins between the lowest and highest.
-            """ % globals()))
+                "Lower threshold", 0, doc="""\
+*(Used only if "%(BC_EVEN)s" selected)*
```

Comment:
Please change to  `*(Used only if using a single measurement and "%(BC_EVEN)s" selected)*`